src/modules/fancyzones/lib/FancyZonesDataTypes.h

```diff
@@ -32,6 +33,25 @@ namespace FancyZonesDataTypes
         Canvas
     };
 
+    struct DeviceIdData
+    {
+        std::wstring deviceName;
+        int width;
+        int height;
+        GUID virtualDesktopId;
+        std::wstring monitorId;
+
+        bool empty() const;
+        std::wstring Serialize() const;
```

Comment:
Maybe to align with deserialization, we can make this static as well with `DeviceIdData` as argument, so we have two utility methods. In addition, `Parse` can be renamed to `Deserialize`.

Reply:
Hm, not sure about that.. This way is unified with json implementation we are using at least

Reply:
However, Serialize is called Stringify there :)